public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -185,7 +146,7 @@ function filterRules(options: PanelProps<UnifiedAlertListOptions>['options'], ru
   return filteredRules;
 }
 
-const getStyles = (theme: GrafanaTheme) => ({
+export const getStyles = (theme: GrafanaTheme) => ({
```

Comment:
can we convert/migrate this to using `GrafanaTheme2` or is that too much hassle?

Reply:
Not all! I think most of the existing props are available under `v1` anyway so will update this 👍